app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
```

Comment:
Isn't this more like `input[type="radio"] + label, input[type="checkbox"] + label`?